backend/presentation/paginations.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework.pagination import PageNumberPagination
+
+
+class PresentationSetPagination(PageNumberPagination):
+    page_size = 10
+    page_size_query_param = "page_size"
+    max_page_size = 100
+
+
+class TagSetPagination(PageNumberPagination):
+    page_size = 10
```

Comment:
You can also specify `DEFAULT_PAGINATION_CLASS` in the settings files. There is no need to create exactly the same classes with different names.